Mention how Siemens applies some patterns (#1)
siemens/InnerSourcePatterns: patterns/2-structured/base-documentation.md

* Mercado Libre - create a documentation site that contains how to get started with InnerSource and also define the basic artifacts that a repository must have to be InnerSource (README, CONTRIBUTING, CODING_GUIDELINES, etc).
* Analog Devices Inc.
* Airbus

> * Siemens is automatically pushing a checklist to every new Inner Source project to make maintainers aware of mandatory requirements as well as best practices.

Reviewer: Not sure if "pushing" implies pushing code/files here. I'd keep it generic like this, or perhaps instead "creates a checklist issue" if we do want to be specific, WDYT?

```suggestion
* Siemens automatically creates a checklist for every new Inner Source project to make maintainers aware of mandatory requirements as well as best practices.
```

```suggestion
* Siemens automatically creates a checklist issue for every new Inner Source project to make maintainers aware of mandatory requirements as well as best practices.
```

Author: I like the version that is explicit about the issue, thanks!
